app/controllers/sessions_controller.rb

```diff
@@ -8,11 +8,11 @@ def new
   end
 
   def create
-    user = User.find_by_email(params[:email])
+    user = User.find_by email: params[:email]
 
     if user && user.authenticate(params[:password])
       session[:user_id] = user.id
-      redirect_to videos_path
+      redirect_to videos_path, notice: "You are signed in, enjoy!"
```

Comment:
Just so you know `alert-notice` is no longer a valid class in bootstrap. It has been changed to `alert-success`. Instead of `alert-error`, it is now `alert-danger`. The helper you are using doesn't support using those names though, so you will have to set the flash keys manually to get the correct display.